mabwiser/base_mab.py

```diff
@@ -394,3 +403,34 @@ def _get_cold_arm_to_warm_arm(self, cold_arm_to_warm_arm, arm_to_features, dista
                 new_cold_arm_to_warm_arm[cold_arm] = closest_arm
 
         return new_cold_arm_to_warm_arm
+
+    def _reset_arm_to_status(self):
+        self.arm_to_status: Dict[Arm, dict] = {arm: {STATUS_TRAINED: False, STATUS_WARM: False,
+                                                     STATUS_WARM_STARTED_BY: None}
+                                               for arm in self.arms}
+
+    def _set_arms_as_trained(self, arms: Optional[List[Arm]] = None, decisions: Optional[np.ndarray] = None,
+                             is_partial: bool = True):
+        """Sets the given arms as trained.
+
+        Uses the ``arms`` if provided, otherwise calculates the arms from the ``decisions``.
```

Comment:
Do you ever use this with arms not None?

Reply:
No, since it's always calculated in LPs using the decisions. However, I could see a custom definition for a neighborhood policy or a new LP which isn't just unique arms in decisions.

Reply:
Maybe remove that bit given that it's not used? Looking to remove some complexity and making it easier to maintain if/when we look at this again.